newsAnalysis/ImagePlotter.py

```diff
@@ -25,7 +25,7 @@ def save(self, path):
         try:
             plt.savefig(path)
         except:
-            print 'Error ImagePlotter: Figure cannot be saved'
+            print('Error ImagePlotter: Figure cannot be saved')
```

Comment:
in contrast to python 2.7, python 3 throws an error for these types of printing messages...
In general, we should agree on the python version =)